src/environments/environment.production.ts

```diff
@@ -11,6 +11,9 @@ export const environment: Environment = {
   getSiteControlId: '97792',
   googleAnalyticsId: 'UA-2979952-1',
   googleOptimizeId: 'OPT-W78W6BT',
+  gg1FunderLogos: [
+    'https://thebiggive--c.visualforce.com/resource/1641902289000/imageGG1FunderExperian',
```

Comment:
@alihejazi I was anticipating that this would just be one composite image so we could avoid new deploys. So I'm not sure we really need the array config or that the file name makes sense.

Reply:
As discussed with Noel via DMs, we canceled the approach of using public URLs because:
1. Static resources aren't available from outside of Salesforce.
2. Public links of image files can't be used in image tags; they don't work because the link runs a form first which then displays a preview of the image. If you refresh the page after loading the public link, you'll see the browser says 'confirm form resubmission?', highlighting what was loaded is a form. The alternative is to right-click on the image that shows after opening that link, and get that url instead, but that url constrains the images to a specific content version, which is no good.